integration_test/README.md

```diff
@@ -148,7 +148,39 @@ Then, inside `applications/<application>/`:
 1.  (if necessary) `exercise`. This is only needed
     sometimes, e.g. to get the application to log to a particular file.
 1.  (if you want to test logging) `expected_logs.yaml`
-1.  (if you want to test metrics) `metric_name.txt`
+1.  (if you want to test metrics) `expected_metrics.yaml`
+
```

Comment:
@jefferbrecht maybe we shall show ppl how to retrieve files like this from google cloud monitoring APIs. A sample script shall suffice. This way observIQ team will know how to generate correct contents in those files

Reply:
Great suggestion, I will add that.

Reply:
This is added now. cc @martijnvans @igorpeshansky since this adds a rather large amount of new lines to review. (generate_expected_metrics.go and the accompanying notes in the README)